spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -56,11 +58,11 @@
     // this ID should be unique for each start of the proxy application, so that CoatRack admin knows when counting was restarted
     private static final String counterSessionID = UUID.randomUUID().toString();
 
-    public void increment(HttpServletRequest request, String apiKey, MetricType metricType, Integer httpResponseCode) {
-        log.debug(String.format("incrementing metric '%s' for URI '%s' and api key %s",
+    public void increment(HttpServletRequest request, String apiKeyValue, MetricType metricType, Integer httpResponseCode) {
+        log.debug(String.format("incrementing metric '%s' for URI '%s' and API key with hash value %s",
```

Comment:
In this place the hash will probably be confusing, as the metric still contains the un-hashed value. This is a general topic to be discussed: how to handle the API key in the metrics.

Reply:
For now, I removed the API key value from the logs. In the future, we will implement a [new metrics system using micrometer](https://github.com/coatrack/coatrack/issues/149) which will handle that and we should shift the discussion to that issue. I already made a comment about this problem.

This comment can be closed.